src/table/table.tsx

```diff
@@ -71,6 +71,8 @@ interface Props<ROW> {
     onPaginate?: (eventType: PaginateEvent, itemsPerPage?: number) => void;
     minRows?: number;
     getRowProps?: (row?: ROW) => object;
+    onSortedChange?: (newSorted: Array<SortingRule>, column: any, shiftKey: any) => void;
```

Comment:
I see issues creating typing for the `column` but `shiftKey`should be boolean, right?